files/en-us/learn/server-side/express_nodejs/deployment/index.html

```diff
@@ -213,8 +221,9 @@ <h3 id="Use_Helmet_to_protect_against_well_known_vulnerabilities">Use Helmet to
 <strong>app.use(helmet())</strong>;
 ...</pre>
 
-<div class="note">
-<p><strong>Note:</strong> The command above adds a <em>subset</em> of the available headers. They make sense for most sites. You can add/disable specific headers as needed by following the instructions on <a href="https://www.npmjs.com/package/helmet">npm</a>.</p>
+<div class="notecard note">
+  <h4>Note</h4>
+  <p>The command above adds a <em>subset</em> of the available headers. They make sense for most sites. You can add/disable specific headers as needed by following the instructions on <a href="https://www.npmjs.com/package/helmet">npm</a>.</p>
```

Comment:
The link text is a bit confusing here. Perhaps it should become
```suggestion
  <p>The command above adds a <em>subset</em> of the available headers. They make sense for most sites. You can add/disable specific headers as needed by following the instructions on <a href="https://www.npmjs.com/package/helmet">helmet</a>.</p>
```
?

Reply:
I modified slightly differently. Thanks.